Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -303,6 +303,9 @@ CClientGame::CClientGame(bool bLocalPlay)
     // Start async task scheduler
     m_pAsyncTaskScheduler = new SharedUtil::CAsyncTaskScheduler(2);
 
+    // Server info construction
+    m_pServerInfo = new CServerInfo();
+
```

Comment:
Also here: Just make it a regular member instead of a pointer. Adding another level of indirection doesn't gain us anything.